workspaces/linux-host/Makefile

```diff
@@ -0,0 +1,66 @@
+# Makefile
+#
+# AUTHORS
+#
+# The Veracruz Development Team.
+#
+# COPYRIGHT AND LICENSING
+#
+# See the `LICENSING.markdown` file in the Veracruz root directory for
+# licensing and copyright information.
+
+.PHONY: all doc clean fmt
+
+PROXY_ATTESTATION_SERVER_DB  = $(abspath proxy-attestation-server.db)
+TEST_PARAMETERS = DATABASE_URL=$(PROXY_ATTESTATION_SERVER_DB) \
```

Comment:
Am I correct in intuiting that the Linux target still has a root enclave? Which is why we still need the `PROXY_ATTESTATION_SERVER_DB`?

Reply:
Yes and yes.